src/Raven.Server/Documents/Replication/Senders/ExternalReplicationDocumentSender.cs

```diff
@@ -20,5 +23,16 @@ protected override TimeSpan GetDelayReplication()
             _parent._parent._server.LicenseManager.AssertCanDelayReplication(external.DelayReplicationFor);
             return external.DelayReplicationFor;
         }
+
+        protected override bool ShouldSkip(DocumentsOperationContext context, ReplicationBatchItem item, OutgoingReplicationStatsScope stats, SkippedReplicationItemsInfo skippedReplicationItemsInfo)
+        {
+            var changeVector = context.GetChangeVector(item.ChangeVector);
+            var changeVectorOrder = changeVector.Order.StripMoveTag(context);
+            item.ChangeVector = changeVector.IsSingle ? 
+                changeVectorOrder : 
+                context.GetChangeVector(changeVector.Version, changeVectorOrder);
```

Comment:
Let's make this functionally in the `ChangeVector` class?

Reply:
done